lib/rules/jsx-sort-default-props.js

```diff
@@ -130,12 +134,6 @@ module.exports = {
             checkSorted(propTypesObject.properties);
           }
           break;
-        case 'CallExpression':
-          const innerNode = node.arguments && node.arguments[0];
```

Comment:
hm, why is this unused exactly?

Could this change be done in a separate commit/PR?

Reply:
See discussion in https://github.com/yannickcr/eslint-plugin-react/issues/1817#issuecomment-396329150
I removed it because coveralls was red and I needed to increase coverage to make it pass 🙂Otherwise I can absolutely make this change separately

Reply:
it'd be great to at least make it separate commits, even if it stays in this PR :-)

Reply:
@ljharb looks like this change isn't necessary for coverage now, so I'll submit a separate PR when this one is in.